<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1041</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Internet Spectral Database:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Compounds List:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>220</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Spectra</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>0</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ramanPi - spectraCide V0.1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QListWidget" name="compoundList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>261</width>
      <height>371</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <item>
     <property name="text">
      <string>Aliphatic compounds</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>alkanes</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>halogenoalkanes</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>alcohols</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>aldehydes and ketones</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>carboxylic acids</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>acyl chlorides</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>acid anhydrides</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>esters</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>amides</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>nitriles</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>amines</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>amino acides etc</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Aromatic compounds</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>arenes</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>aryl halides</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>phenol</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>phenylamine</string>
     </property>
    </item>
   </widget>
   <widget class="QListWidget" name="internetSpectralDatabaseList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>261</width>
      <height>201</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <item>
     <property name="text">
      <string>RRUFF </string>
     </property>
    </item>
   </widget>
   <widget class="QGraphicsView" name="spectralView">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>240</y>
      <width>741</width>
      <height>411</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="quit_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>620</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>741</width>
      <height>201</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QDial" name="exposureDial">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>101</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>6</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="notchesVisible">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="sampleName_box">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>110</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>ramanPi_</string>
     </property>
     <property name="maxLength">
      <number>128</number>
     </property>
    </widget>
    <widget class="QProgressBar" name="captureProgress">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>160</y>
       <width>591</width>
       <height>23</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>User Key</string>
     </property>
    </widget>
    <widget class="QPushButton" name="getSpectra_btn">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>20</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Get Spectra</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="userName_box">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>YOUR_USER_ID</string>
     </property>
     <property name="maxLength">
      <number>50</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="userKey_box">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>YOUR_USER_KEY</string>
     </property>
     <property name="maxLength">
      <number>25</number>
     </property>
    </widget>
    <widget class="QPushButton" name="popUpSpectra_btn">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>60</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>PopUp Spectra</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="exposureLCD">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="digitCount">
      <number>1</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
     <property name="value" stdset="0">
      <double>1.000000000000000</double>
     </property>
     <property name="intValue" stdset="0">
      <number>1</number>
     </property>
    </widget>
    <widget class="QPushButton" name="postSpectra_btn">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>20</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Share Spectra</string>
     </property>
    </widget>
    <widget class="QComboBox" name="postSite_combo">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>20</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>plot.ly</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>phant</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>User Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>CCD Exposure Setting</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="printSpectra_btn">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>100</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Print Spectra</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saveSpectra_btn">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>60</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Save Spectra</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>90</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Sample Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>140</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Capture Progress</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ramanPi IP Address</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>10.0.0.254</string>
     </property>
    </widget>
    <widget class="QPushButton" name="connect_btn">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>110</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect to ramanPi</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="connected_ind">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>80</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Connected</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="clearSpectra_btn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>620</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>clearSpectra</string>
    </property>
   </widget>
   <zorder>frame</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_7</zorder>
   <zorder>compoundList</zorder>
   <zorder>internetSpectralDatabaseList</zorder>
   <zorder>spectralView</zorder>
   <zorder>quit_btn</zorder>
   <zorder>clearSpectra_btn</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1041</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRamanPi_spectraCide_v0_1">
    <property name="title">
     <string>ramanPi_spectraCide_v0.1</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuRamanPi_spectraCide_v0_1"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>exposureDial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>exposureLCD</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>345</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
